### PR TITLE
WIP 123084 Provide a diff for the provisioning script

### PR DESCRIPTION
**Pull request by zagy**

### CODE REVIEW

File: src/batou_ext/fcio.py [hunk 1]
```diff
@@ -71,39 +72,34 @@ def configure(self):
     def _compute_calls(self):
         assert self.project
         self.aliases = []
-        for host in self.environment.hosts.values():
+        for host in self.environment_.hosts.values():
```
Comment (frlan) [inline src/batou_ext/fcio.py]: Looks like a typo for me -- _ intentionally?

Comment (zagy) [reply src/batou_ext/fcio.py]: Yes,  it's a  component, hence environment is already used by batou.

File: src/batou_ext/fcio.py [hunk 2]
```diff
@@ -152,70 +146,51 @@ class Provision(batou.component.Component):
     vm_environment = "fc-15.09-production"
     api_url = "https://{project}:{api_key}@api.flyingcircus.io/v1"
 
-    @staticmethod
-    def load_env(env_name):
-        environment = batou.environment.Environment(env_name)
+    def __init__(self, env_name=None, dry_run=False, diff=False, **kwargs):
```
Comment (frlan) [inline src/batou_ext/fcio.py]: Missing super()-call?

Comment (zagy) [reply src/batou_ext/fcio.py]: Fixed.

**Pull request merged**